frontend/src/components/ScheduledEventsTableCard/ScheduledEventsTableCard.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import axios from 'axios';
+import { Card } from 'react-bootstrap';
+import { Dimmer, Loader } from 'semantic-ui-react';
+import { ScheduledEventsTable } from '../ScheduledEventsTable/ScheduledEventsTable';
+import { api, localForageKeys } from '../../utils';
+import * as localForage from 'localforage';
+import { PaginationRow } from '../PaginationRow';
```

Comment:
There's a nice feature of VSCode that arranges imports `Command + Shift + P` -> `Organise Imports`